include/zephyr/bluetooth/audio/cap.h

```diff
@@ -945,14 +956,26 @@ int bt_cap_commander_broadcast_reception_start(
 	const struct bt_cap_commander_broadcast_reception_start_param *param);
 
 /** Parameters for stopping broadcast reception  */
+
+struct bt_cap_commander_broadcast_reception_stop_member_param {
+	/** Coordinated or ad-hoc set member. */
+	union bt_cap_set_member member;
+
+	/** Source ID of the receive state. */
+	uint8_t src_id;
+
+	/** Number of subgroups */
+	size_t num_subgroups;
```

Comment:
I think the description of this needs improvement. 

IIRC this value shall be the reported number of subgroups in the specified receive state, right? (At least until we may see a fix in a future version of BASS).

That should be added here.